server/src/main/java/org/eclipse/openvsx/adapter/VSCodeAdapter.java

```diff
@@ -397,6 +404,10 @@ private String getVscodeEngine(ExtensionVersion extVer) {
                 .orElse(null);
     }
 
+    private boolean isWebExtension(ExtensionVersion extVer) {
+        return extVer.getExtensionKind() != null && extVer.getExtensionKind().contains("web");
+    }
```

Comment:
It can be seen as an invariant that an extension is a web extension, if and only if the extension kind contains 'web'.